scripts/bootstrap.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+for i in "$@"
+do
+case $i in
+    --vault_addr=*)
+    vault_addr="${i#*=}"
+    shift # past argument=value
+    ;;
+    --vault_role=*)
+    vault_role="${i#*=}"
+    shift # past argument=value
+    ;;
+    --env=*)
+    env="${i#*=}"
+    shift # past argument=value
+    ;;
+    --epoch_package=*)
+    epoch_package="${i#*=}"
+    shift # past argument=value
+    ;;
+    --default)
+    DEFAULT=YES
+    shift # past argument with no value
+    ;;
+    *)
+          # unknown option
+    ;;
+esac
+done
+
+cd $(dirname $0)/../ansible/
+# Temporary fix/workaround for non-executable vault install
+chmod +x /usr/bin/vault
+
+# Authenticate the instance to CSM
+PKCS7=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7 | tr -d '\n')
+
+export VAULT_ADDR=$vault_addr
+export VAULT_TOKEN=$(vault write -field=token auth/aws/login pkcs7=$PKCS7 role=$vault_role)
+
+export env=$env
```

Comment:
why exporting this?

Reply:
i will check is it working without... but... good question